Dockerfile

```diff
@@ -0,0 +1,70 @@
+
+#Use debian:stable-slim as a builder and then copy everything.
+FROM debian:stable-slim as builder
+
+#Set mosquitto and plugin versions.
+#Change them for your needs.
+ENV MOSQUITTO_VERSION=1.6.3
```

Comment:
Could you bump Mosquitto and Go versions to latest?

Reply:
Done.